### PR TITLE
Use correct SARI & request id

### PR DESCRIPTION
**Pull request by epinxteren**

### CODE REVIEW

File: src/Logger/AuthnRequestIdLoggerDecorator.php [hunk 1]
```diff
@@ -22,9 +22,9 @@
 use Surfnet\GsspBundle\Service\StateHandlerInterface;
 
 /**
- * Add's stepup request id to the logs.
+ * Add auth request id to the logs.
  */
-final class StepupRequestIdLoggerDecorator extends AbstractLogger
+final class AuthnRequestIdLoggerDecorator extends AbstractLogger
```
Comment (jorissteyn) [inline src/Logger/AuthnRequestIdLoggerDecorator.php:27]: Almost there! This logger logs sari, not the request ID of the SAML AuthnRequest.

    $this->stateHandler->getStepupRequestId()

vs

    $this->stateHandler->getRequestId()

**Review by jorissteyn — CHANGES_REQUESTED** [review]
Another question, where is StateHandler::setStepupRequestId() and StateHandler::setRequestId() called in gssp-bundle / example and tiqr?

File: src/Logger/AuthnRequestIdLoggerDecorator.php (hunk 1 above)
Comment (epinxteren) [reply src/Logger/AuthnRequestIdLoggerDecorator.php:27]: Fixed this

**Comment by epinxteren** [comment]
@jorissteyn 

Both setters are called in saveRegistrationRequest and saveAuthenticationRequest of the StateHandler

**Pull request merged**